### ISSUE TITLE
Scheduled restart an hour early

### ISSUE THREAD

**Issue by phelpsiv** [matt-cook/kiosk#151]
@matt-cook I just wanted to let you know that browser.js is setting the app to restart an hour earlier than what's scheduled on line 119.
```
var hour = parseInt(data.restart) - 1;
```

should be:
```
var hour = parseInt(data.restart);
```

**Comment by matt-cook** [comment]
Thanks for the report!

**Comment by RabsRincon** [comment]
Ey! Revising the issues to find something to do, I found this one, and I propose to close this issue as it's **not** a bug. 
moment.js's `hour()` method accepts a range from **[0, 23]** ([as mentioned on its docs](https://momentjs.com/docs/#/get-set/hour/)) while `data.restart` is **[1, 24]**.

**Comment by phelpsiv** [comment]
If that's the case I'd suggest at very least mentioning this in the read me. I'd argue that if moment.js is how time is managed in this app it would prevent confusion to have data.restart work on the same [0,23] convention as this leads to confusion and prevents the app from being intuitive. I'd also argue it is a bug as when setting the time in the interface to restart at 1pm it always restarts at noon, which I don't believe is it's intended functionality.

**Comment by jonfairman** [comment]
Just chiming in that I've observed this behavior as well. I setup a Chromebit in Sydney, Australia and if I set the restart time to 4PM, it restarts at 3PM instead (or whatever time, but an hour earlier). Not sure if this is related to daylight savings time or what, but figured I'd add another report ¯\_(ツ)_/¯.

**Comment by matt-cook** [comment]
This issue has been fixed in v5.13.0 (https://github.com/matt-cook/kiosk/pull/256) which was just published in the app store.
